src/modules/lua-support/mtev/Api.lua

```diff
@@ -0,0 +1,188 @@
+--
+-- mtev.API
+--
+-- Wrapper class for HTTP/HTTPS APIs
+--
+module("mtev.API", package.seeall)
+
+local HttpClient = require('mtev.HttpClient')
+
+local function HTTP(method, host, port, uri, headers, payload, sslconfig)
+  assert(method)
+  assert(host)
+  assert(port)
+  local ssl = not not sslconfig
```

Comment:
`not not` is odd here .. how about `local ssl = sslconfig ~= nil` ?

Reply:
`not not x` is an idiom from converting x to bool.

Reply:
yes as discussed, this handles an accidental `false` here, all good